Makefile

```diff
@@ -11,11 +11,11 @@ build-externals-extra:
 	@ mkdir -p  ./vault/data ./vault/policies
 	@ chmod 777  ./vault/data ./vault/policies
 
-start-elk: build-externals
+start-elk: build-externals import-kibana-dashboards
```

Comment:
import & export shall not be linked to start/stop services; let them be independent (but I appreciate the intention of automating that).